### ISSUE TITLE
Publish to pypi

### ISSUE THREAD

**Issue by IlyaSemenov** [karanlyons/django-save-the-change#22]
The pypi version is almost 3 years old, can you please update it?

**Comment by karanlyons** [comment]
There’ve been a whole bunch of changes that I’ve got no feedback on, hence my reluctance to release the version onto PyPI. As far as I know there aren’t any problems with the current version, but my tests and running it in prod are just two datapoints.

Tell you what: If you can run HEAD and don’t see any issues on your end, I’ll push it as a true release. But I don’t want to end up pushing a broken release and get a whole bunch of issues opened up asking why I didn’t test things more thoroughly.

**Comment by mdamien** [comment]
Current version on Pypi triggers an import error with django 1.10:

        from save_the_change.mixins import SaveTheChange
      File "save_the_change/mixins.py", line 11, in <module>
        from django.db.models.related import RelatedObject
    ImportError: No module named 'django.db.models.related'

But the latest `master` version does not, I think it's time to publish the new version

**Comment by karanlyons** [comment]
Yeah, honestly I’m just going to publish the thing at this point. I can almost guarantee that doing so will lead to *more* bug reports, which I’d much rather get before pushing updates, but eh, die is cast.

**Comment by mdamien** [comment]
+1 and at least you're gonna get good bug reports (against master instead of against an old version)

**Comment by oTree-org** [comment]
+1, my project uses SaveTheChange pretty heavily and all my tests are passing.

**Comment by anirvan-majumdar** [comment]
+1 - @karanlyons please publish the update to pypi
